tests/hypothesis_strategies/kernels.py

```diff
@@ -2,13 +2,29 @@
 
 import hypothesis.strategies as st
 
-from baybe.kernels import MaternKernel
+from baybe.kernels import MaternKernel, ScaleKernel
 
 from ..hypothesis_strategies.priors import priors
 
 matern_kernels = st.builds(
     MaternKernel,
     nu=st.sampled_from((0.5, 1.5, 2.5)),
-    lengthscale_prior=st.one_of(st.none(), priors),
+    lengthscale_prior=priors,
```

Comment:
why did you now get rid of the `None` option?

Reply:
Reintroduced, that was just a mistake.